components/Edit.jsx

```diff
@@ -26,13 +44,13 @@ var MarkdownEditor = React.createClass({
       return;
     }
     $.post(
-      '/article/'+name+'/edit',
+      '/article/' + this.getParams().id + '/edit',
       {name: name, text: text},
       function(data) {
-        console.log(JSON.stringify(data));
+        // redirect to article page post-edits
+        this.transitionTo('article', {id: this.getParams().id});
```

Comment:
adding a Router.Navigation mixin, one should be able to "transitionTo" the newly-edited article (i.e redirect)